utilities/db_util.py

```diff
@@ -0,0 +1,41 @@
+from dotenv import dotenv_values
+import mysql.connector
+
+def connect_to_db(db: str=""):
+  config = dotenv_values(".env")
+  try:
+    if len(db)==0:
+      mydb = mysql.connector.connect(
+        host="localhost",
+        user=config['USER'],
+        password=config['PASSWORD'],
+      )
+    else:
+      mydb = mysql.connector.connect(
+        host="localhost",
+        user=config['USER'],
+        password=config['PASSWORD'],
+        database=db,
+      )
+    return mydb
+  except Exception as e:
+    print(f"Exception happend while connecting to db:{e}")
+    return None
+
+def execute_query(query : str,cursor):
+  try:
+    result = cursor.execute(query)
+    return result
+  except Exception as e:
+    print(f"Failure while executing query [{query}]:{e}")
+    return None
+
+def check_if_exists(cursor, db : str, query) -> bool:
```

Comment:
what is this method checking exactly ? 
Name `check_if_exists` seems too general for something only db related

Reply:
Gotcha, maybe we should have two separate methods for this, since we know query in both situations
```
check_if_table_exists(cursor, table_name: str) -> bool:
    return _check_if_exists(cursor, table_name, "SHOW TABLES")


check_if_database_exists(cursor, db_name: str) -> bool:
    return _check_if_exists(cursor, db_name, "SHOW DATABASES")
```

We just prefix `_` to the name of `check_if_exists` so that it's clear that it is our internal method not meant to be publicly used, and that we should use two methods above

Reply:
Just one more thing. I think that this whole method could be written like this:
```
cursor.execute(query)
return any([db == item[0] for item in cursor.fetchall()])
```

Reply:
This last line of code works awesome.